paypal-commercetools-events/tests/event.spec.ts

```diff
@@ -44,7 +38,7 @@ describe('Testing PayPal-commercetools-events Controller', () => {
         .mockReturnValueOnce({
           body: {
             value: {
```

Comment:
"value: { sendTrackingToPayPal }," should be okay as well since the variable name matches the field name